python/buildDcrModel.py

```diff
@@ -0,0 +1,616 @@
+"""Attempts to create airmass-matched template images for existing images in an LSST repository."""
+
+# LSST Data Management System
+# Copyright 2016 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+from __future__ import print_function, division, absolute_import
+
+import numpy as np
+from scipy.ndimage.interpolation import shift as scipy_shift
+
+import lsst.afw.image as afwImage
+import lsst.afw.math as afwMath
+import lsst.meas.algorithms as measAlg
+
+from .dcr_utils import wrap_warpExposure
+from .dcr_utils import solve_model
+from .generateTemplate import GenerateTemplate
+
+__all__ = ["BuildDcrModel"]
```

Comment:
Looks like I have the older flake8, so I'll leave these for now but switch it once I've updated my linter.